packages/loader/container-loader/src/loader.ts

```diff
@@ -243,8 +245,15 @@ export interface ILoaderServices {
      * The logger downstream consumers should construct their loggers from
      */
     readonly subLogger: ITelemetryLogger;
+
+    readonly detachedBlobStorage?: IDetachedBlobStorage;
 }
 
+ export interface IDetachedBlobStorage {
+     createBlob(content: ArrayBufferLike): Promise<ICreateBlobResponse>;
+     readBlob(id: string): Promise<ArrayBufferLike>;
+ }
+
```

Comment:
Consider defining IDetachedBlobStorage as a subset of the IDocumentStorageService:
```ts
export interface IDetachedBlobStorage implements Pick<Partial<IDocumentStorageService>, 'createBlob' | 'readBlob'> { }
```